### DISCUSSION TITLE
How can we contact with mentors??

### DISCUSSION THREAD

**Discussion by sourav8927** [camicroscope/GSOC#24] [Q&A]
How can we contact with mentors bcs on the list of ideas there given the mentors name not given their contact mail id or anything ,
bcs i want to work on the project collection and study managment and modularization of components ,

**Comment by pranavvp16** [comment]
My contact mentor is Tony Pan, but I'm unable to find him on slack as well as on this discussion section

**Reply by tcpan** [reply]
Hi, @pranavvp16,
Sorry for the delay in responding.   I am not using Slack.  Please let me know if you have any specific questions.

Tony

**Reply by pranavvp16** [reply]
If you can help me with the proposal for  task 1 Machine learning assistant for GSOC. Any draft or previous proposal would also work. Please if you can provide me with it ASAP so I can submit my proposal as the deadline for proposal submission is 4th april

**Reply by tcpan** [reply]
@ pranavvp16, I don't have any previous or draft proposal that I can share.  For format, please follow the organization described at the end of the landing page of this repo.  For technical content, I would suggest to discuss your code challenge response in the proposal, include as detailed an implementation plan as reasonable, and identify potential problems (risks) you may encounter and alternatives..

Hope that helps.  Please let me know if you have more questions.

**Comment by sourav8927** [comment]
my mentor is Nan li and exactly same thing happened to me but still i
cant find any solution

**Reply by nanli-emory** [reply]
Hi @sourav8927, sorry for the delayed response. We're receiving a high volume of very general questions. Please submit your proposal with the code challenge and I will review it ASAP. Thank.

**Comment by birm** [comment]
The discussion post based system is new to us this year. It might be better if you post specific questions or ideas you have.
I can say that personally, I'm trying to avoid excessive one on one communication before contributor selection, and am trying to promote open communication as much as possible.

**Comment by sourav8927** [comment]
thank  you , I appreciate 😊.
